pilot/pkg/networking/core/v1alpha3/listener.go

```diff
@@ -1912,12 +1912,14 @@ func buildListener(opts buildListenerOpts, trafficDirection core.TrafficDirectio
 				}
 			}
 			if !fullWildcardFound {
-				sort.Strings(chain.sniHosts)
+				chain.sniHosts = append([]string{}, chain.sniHosts...)
```

Comment:
what effect does this have? looks like its appending a slice to an empty slice and assigning it to the original slice